### PR TITLE
Bricked export

### PR DESCRIPTION
**Pull request by adewit**
This adds a new bricked module type so the `isBricked` property can be written into the XML's. Nothing changes in the TkLayout analysis (no new resolutions etc.), at this point really just to make it easier to set the property in the XML's correctly. 

No configs included in the PR for now, but an example geometry can be found here: https://adewit.web.cern.ch/adewit/layouts/OT800_IT700_BRICKED/info.html

### CODE REVIEW

File: include/Sensor.hh [hunk 1]
```diff
@@ -52,6 +53,7 @@ public:
     stripLengthEstimate("stripLengthEstimate", parsedOnly()),
     numROCX("numROCX", parsedOnly()),
     numROCY("numROCY", parsedOnly()),
+    isBricked("isBricked", parsedOnly(),false),
```
Comment (ghugo83) [inline include/Sensor.hh:56]: spacing here

File: src/XMLWriter.cc [hunk 2]
```diff
@@ -966,6 +966,9 @@ namespace insur {
       if( param.second.find("TOBDet") != std::string::npos) param.second = xml_subdet_tobdet_1;
       if( param.second.find("TIDDet") != std::string::npos) param.second = xml_subdet_tiddet;
       stream<< xml_spec_par_parameter_first << xml_tkddd_structure << xml_spec_par_parameter_second  << param.second  << xml_general_endline;
+      if( minfo.at(i).bricked ){ // Only add the line if isBricked exists
+          stream<< xml_spec_par_parameter_first << xml_bricked_name    << xml_spec_par_parameter_second << xml_true <<xml_general_endline;
```
Comment (ghugo83) [inline src/XMLWriter.cc:970]: and here,

otherwise looks ok :)

Comment (adewit) [reply src/XMLWriter.cc]: Ok, hope it's better now!

**Comment by ghugo83** [comment]
Great, yes it does
@alkemyst ok if I merge it?

**Comment by alkemyst** [comment]
Totally


---- ghugo83 wrote ----


Great, yes it does
@alkemyst<https://github.com/alkemyst> ok if I merge it?

**Pull request merged**